qiskit/providers/models/backendconfiguration.py

```diff
@@ -400,16 +401,41 @@ def acquire(self, qubit: int) -> AcquireChannel:
             raise BackendConfigurationError("Invalid index for {}-qubit systems.".format(qubit))
         return AcquireChannel(qubit)
 
-    def control(self, channel: int) -> ControlChannel:
+    def control(self, qubit: int) -> ControlChannel:
         """
         Return the secondary drive channel for the given qubit -- typically utilized for
         controlling multiqubit interactions. This channel is derived from other channels.
 
         Returns:
             Qubit control channel.
         """
-        # TODO: Determine this from the hamiltonian.
-        return ControlChannel(channel)
+        return ControlChannel(qubit)
```

Comment:
Won't the control channels have keys we could use here to map to the info provided in the latest schema update?

Reply:
I talked about this offline and should be fixed by [8cb4551](https://github.com/Qiskit/qiskit-terra/pull/4097/commits/8cb4551cd4ae9cd0a9733affb35355c3b9f19d21)